### PR TITLE
Remove convert_tokenizer

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: .github/workflows/lcm_dreamshaper_cpp.yml [hunk 1]
```diff
@@ -101,9 +98,7 @@ jobs:
         working-directory: ${{ env.working_directory }}
         run: |
           conda activate openvino_lcm_cpp
-          $env:MODEL_PATH='models/lcm_dreamshaper_v7/FP16'
-          optimum-cli export openvino --model SimianLuo/LCM_Dreamshaper_v7 --weight-format fp16 $env:MODEL_PATH
-          convert_tokenizer $env:MODEL_PATH/tokenizer/ --tokenizer-output-type i32 -o $env:MODEL_PATH/tokenizer/
```
Comment (ilya-lavrenov) [inline .github/workflows/lcm_dreamshaper_cpp.yml:106]: tokenizer conversion is not implemented for ClipTokenizer now
CVS-136848

File: .github/workflows/causal_lm_cpp.yml [hunk 2]
```diff
@@ -33,10 +33,9 @@ jobs:
           optimum-cli export openvino --trust-remote-code --weight-format fp16 --model openlm-research/open_llama_3b_v2 open_llama_3b_v2
           cmake -DCMAKE_BUILD_TYPE=Release -S ./text_generation/causal_lm/cpp/ -B ./build/
           cmake --build ./build/ --config Release -j
-      - name: convert_tokenizer and run
+      - name: greedy_causal_lm
         run: |
           source ./ov/setupvars.sh
-          convert_tokenizer ./open_llama_3b_v2/ --output ./open_llama_3b_v2/ --with-detokenizer
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:39]: looks like newer optimum version is required 
image

File: .github/workflows/lcm_dreamshaper_cpp.yml (hunk 1 above)
Comment (ilya-lavrenov) [reply .github/workflows/lcm_dreamshaper_cpp.yml:106]: Looks like https://github.com/huggingface/optimum-intel/pull/697 can help

File: text_generation/causal_lm/cpp/speculative_decoding_lm.cpp [hunk 3]
```diff
@@ -119,10 +119,10 @@ int main(int argc, char* argv[]) try {
     core.add_extension(OPENVINO_TOKENIZERS_PATH);  // OPENVINO_TOKENIZERS_PATH is defined in CMakeLists.txt
     // tokenizer and detokenizer work on CPU only
     ov::InferRequest tokenizer = core.compile_model(
-        std::string{argv[1]} + "/openvino_tokenizer.xml", "CPU").create_infer_request();
+        std::string{argv[1]} + "/openvino_tokenizer/openvino_tokenizer.xml", "CPU").create_infer_request();
     auto [draft_input_ids, draft_attention_mask] = tokenize(tokenizer, argv[3]);
     ov::InferRequest detokenizer = core.compile_model(
-        std::string{argv[1]} + "/openvino_detokenizer.xml", "CPU").create_infer_request();
+        std::string{argv[1]} + "/openvino_tokenizer/openvino_detokenizer.xml", "CPU").create_infer_request();
```
Comment (ilya-lavrenov) [inline text_generation/causal_lm/cpp/speculative_decoding_lm.cpp:125]: @apaniukov 
why extra subfolder is created? and why is it names `openvino_tokenizer` not just `tokenizer` like here https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7/tree/main/tokenizer ?

Comment (apaniukov) [reply text_generation/causal_lm/cpp/speculative_decoding_lm.cpp:125]: `tokenizer` and `tokenizer_2` are from a stable diffusion repo structure. The `openvino_tokenizer` folder is just an arbitrary name: https://github.com/huggingface/optimum-intel/pull/580#discussion_r1557865119, can be changed at any time.

Comment (ilya-lavrenov) [reply text_generation/causal_lm/cpp/speculative_decoding_lm.cpp:125]: I would keep folders structure as in HF

**Pull request merged**